add Windows Terminal BY Stevens Antony

stevensantonygit opened the pull request:
# TerminalCraft Project Submission

## Project Details

1. **What is the name of your project?**
Windows Terminal Project
2. **How many hours did you spend developing this?**
   _(Give your best estimate or show proof using hackatime/wakatime)_
I spent 7+ hours on this but did not track the time. But project work is worth 7+ hrs.
3. **Briefly describe what your project is. What makes it unique?**
   _(Provide a brief description of the functionality or innovation)_
It is like a Windows terminal with a owo game and a funny exiting animation and many more things to see and try out!
4. **How many people have tested your extension?**
   _(We recommend at least 3)_
Many of my friends and family members have tried it! Around 14+ people.
5. **On What operating systems have you tested this project?**
   _(MacOS/Linux/Windows)_
MacOS/Linux/Windows. Literally all!
6. **Does your project include a Readme file with instructions on how to use it?**
   _(Yes/No)_
Yes
---

## Additional Comments
I have send you in the DMs in slack for the different files of the terminal code and info and also screenshots on how it works!
Hope you like it!
My slack id is U08260D58NN. My name is Stevens Antony.
Signing off....

JosiasAurel reviewed (changes requested):
This has to be a program that runs in an actual terminal and not a web app

stevensantonygit commented:
Check dms. I cant add because of github storage problem! I have sent the
terminal application in slack Dms.

On Wed, 19 Mar 2025 at 11:47, Josias ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> This has to be a program that runs in an actual terminal and not a web app
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/terminalcraft/pull/103#pullrequestreview-2696822300>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQTPGPFZU2F7Y6MKUES4MFL2VDSFTAVCNFSM6AAAAABZIWHNDGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMOJWHAZDEMZQGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

JosiasAurel reviewed (changes requested):
Delete the `node_modules` folder.

Also the games don't have a loop. Try putting in a bit of structure there

stevensantonygit commented:
Fixed the structure but cant delete node module as it is required to run
the programs. When i delete node modules, and run the programs "node
terminal.js"...It will not run showing an error. Thus the node
modules should be there and paths for the file cannot be found. But the
terminal.js program i have fixed and updated it here at
https://github.com/stevensantonygit/terminalcraft

On Wed, 19 Mar 2025 at 20:48, Josias ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Delete the node_modules folder.
>
> Also the games don't have a loop. Try putting in a bit of structure there
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/terminalcraft/pull/103#pullrequestreview-2698299220>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQTPGPAGCQQOZJ5LETA4EE32VFRQFAVCNFSM6AAAAABZIWHNDGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMOJYGI4TSMRSGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

JosiasAurel reviewed (changes requested):
This doesn't really do anything does it? Nor the games have a proper game loop

stevensantonygit commented:
What do you mean? I have send you the video of the it in slack dms! long
ago but you did not check. Pls check. Everything works even the loops...

On Thu, 20 Mar 2025 at 22:21, Josias ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> This doesn't really do anything does it? Nor the games have a proper game
> loop
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/terminalcraft/pull/103#pullrequestreview-2702943554>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQTPGPDDYFPS4JBVP44UEV32VLFH5AVCNFSM6AAAAABZIWHNDGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDOMBSHE2DGNJVGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
